app/assets/javascripts/application/capacity/index.js

```diff
@@ -98,7 +97,7 @@ function transform(){
 
 function updateGraph(){
   dataCall = d3.xhr("http://localhost:3000/capacity/get")
-  params = {id: 1}
+  params = {capacity_id: 1}
```

Comment:
This was hardcoded in. Should default to letting the controller handle which capacity to change.